### ISSUE TITLE
The nodejs-npm is no longer existed in latest alpine

### ISSUE THREAD

**Issue by anguschang007** [nigelpoulton/psweb#10]
I am getting the error while I am trying to build the image from your Dockerfile. However, I found the issue is coming from the nodejs-npm that is no longer existed in the latest alpine.

I change the nodejs-npm to npm now then all is going fine.

> Sending build context to Docker daemon  66.56kB
Step 1/8 : FROM alpine
latest: Pulling from library/alpine
5843afab3874: Pull complete 
Digest: sha256:234cb88d3020898631af0ccbbcca9a66ae7306ecd30c9720690858c1b007d2a0
Status: Downloaded newer image for alpine:latest
 ---> d4ff818577bc
Step 2/8 : LABEL maintainer="nigelpoulton@hotmail.com"
 ---> Running in 62f961bd649a
Removing intermediate container 62f961bd649a
 ---> a01ed7dfb8f0
Step 3/8 : RUN apk add --update nodejs nodejs-npm
 ---> Running in d8bda7e3eb67
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
ERROR: unable to select packages:
  nodejs-npm (no such package):
    required by: world[nodejs-npm]
The command '/bin/sh -c apk add --update nodejs nodejs-npm' returned a non-zero code: 1

**Comment by Createitv** [comment]
I have the same problem

**Comment by nikmar13** [comment]
Yes, I have the same issue. You can change the name of the package `nodejs-npm` to `npm`. And then you can build the image!

**Comment by mudspringhiker** [comment]
Also got the same problem and did the same thing.

**Comment by nigelpoulton** [comment]
Thanks everyone. All sorted now.

**Comment by RickyChiu1231** [comment]
thx bro

**Comment by SyedZain110** [comment]
Thanks a lot ...i have a same issue..but solved now

**Comment by faltaie** [comment]
I ran Dockerfile in GitHub (copy & paste) and I got the following error:
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /src/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/src/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-04-26T07_04_50_562Z-debug.log
